src/java.base/share/classes/java/lang/constant/ClassDesc.java

```diff
@@ -49,18 +49,15 @@
  * {@linkplain ClassDesc} for the component type and then call the {@link #arrayType()}
  * or {@link #arrayType(int)} methods.
  *
- * @apiNote In the future, if the Java language permits, {@linkplain ClassDesc}
- * may become a {@code sealed} interface, which would prohibit subclassing except
- * by explicitly permitted types.  Non-platform classes should not implement
- * {@linkplain ClassDesc} directly.
- *
  * @see ConstantDescs
  *
  * @since 12
  */
-public interface ClassDesc
+sealed public interface ClassDesc
```

Comment:
Should move `sealed` behind `public` per the blessed modifier order from JLS. https://docs.oracle.com/javase/specs/jls/se16/preview/specs/sealed-classes-jls.html#jls-8.1.1

Per http://openjdk.java.net/jeps/409 the finalized sealed classes have no difference from that in 16, so the order suggested there should be valid.

Reply:
Thanks. Now changed.